sql/branch.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+    UPDATE payment_instructions SET due = 0 WHERE due > 9.41;
```

Comment:
I'm outta time to work on this today, though. I will try to pick up again tomorrow, @rohitpaulk.